### DISCUSSION TITLE
Lost text sharpness /clarity after migration from MD 3.1 (on NET FW 4.8) to MD 5.0 (on NET 8)!

### DISCUSSION THREAD

**Discussion by DrCQ** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3538] [Q&A]
I developed two years ago an application on NET Framework 4.8 using the Material Design 3.1 package. Now I'm porting this solution to NET 8 with Material Design 5.0 package. It's a lot of work, and I try to create some documentation in parallel. Now I faced a big problem while comparing the results. In the new version, the text sharpness/clarity is lost.

<img width="450" alt="MD 3 1 vc 5 0" src="https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/assets/49019093/ba16227f-26e4-438a-b81b-d79e25c6660a">

On the left site is the 'old' and on the right site the 'new' application. I have no idea where to look for.
I checked all XAML settings for the `MainWindow`, tested all settings for `TextOptions` and `RenderOptions` without success. 

Any idea what is happening here?

**Comment by Keboo** [comment]
That looks a like a [ClearType](https://learn.microsoft.com/dotnet/desktop/wpf/advanced/cleartype-overview?view=netframeworkdesktop-4.8&WT.mc_id=DT-MVP-5003472) issue. You will find docs that talk about registry settings, these are **not** what you are looking for.

The most likely cause is things like Opacity, or a DropShadowEffect somewhere in the UI hierarchy as this will disable ClearType. Inside of this library look for things settings `ElevationAssist` as this attached property is just a shortcut for adding a drop shadow.

**Reply by DrCQ** [reply]
Hi, to find the problem, I created a short app with one `StatusBar` control. One instance is shown directly, and the other one is located in the `Card` control. The problem is clear for me: the `Card` control makes the whole content blurry. 

![CardSnapshot-250](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/assets/49019093/0d18dd47-0070-4e41-aeec-878a99a1a97b)

I found the `Card` template settings for `ElevationAssist` but setting DP0 don't solve the problem.
Any idea from the expert?

Regards,

Christoph

**Reply by DrCQ** [reply]
Hi, I just continued with my investigations and tested also the `MaterialDesignOutlinedCard` style. The result is much better than for the `MaterialDesignElevatedCard` (which seems to be the default style). 

<img width="778" alt="Snap outlined-plain-elevated" src="https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/assets/49019093/c8efbf43-4832-433a-af95-a0ad00a802db">

This snapshot shows the `StatusBar `control in Outlined Card (top), plain and in Elevated Card (bottom). So I will try to use this _Outlined_  style for my project.

Thanks,
Christoph

**Comment by DrCQ** [comment]
Hi @Keboo,

as mentioned above, my problem is solved. My impression is, that something is wrong with Elevated Card Style because the content is blurry.

Thanks,
Christoph

**Reply by Keboo** [reply]
Thanks for the update, I am still thinking this is a bug with the Elevated Card and will investigate.

**Reply by Keboo** [reply]
After some investigation, it looks like this is probably the same as issue #3499. Can you test with the latest preview package and see if the issue is resolved?
